PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -181,8 +198,21 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
   static constexpr char NameHistPosYAfterEvSel[] = "hPosYAfterEvSel";
   static constexpr char NameHistNumPvContributorsAfterSel[] = "hNumPvContributorsAfterSel";
   static constexpr char NameHistCollisionsCentOcc[] = "hCollisionsCentOcc";
-
-  std::shared_ptr<TH1> hCollisions, hSelCollisionsCent, hPosZBeforeEvSel, hPosZAfterEvSel, hPosXAfterEvSel, hPosYAfterEvSel, hNumPvContributorsAfterSel;
+  static constexpr char NameHistUPC[] = "hUPCollisions";
+
+  // upc preselection
+  static constexpr int MinNdtcoll = 1;       // Default number of sigma (NDtcoll)
+  static constexpr int MinNBCs = 2;          // Minimum number of bunch crossings (NBCs)
+  static constexpr int MinNTracks = 2;       // Minimum number of PV contributors
+  static constexpr int MaxNTracks = 1000;    // Maximum number of PV contributors
+  static constexpr float MaxFITtime = 34.f;  // Maximum FIT time in ns
+  static constexpr float FITAmpFV0 = -1.f;   // FV0 amplitude
+  static constexpr float FITAmpFT0A = 150.f; // Max FT0A amplitude
+  static constexpr float FITAmpFT0C = 50.f;  // Max FT0C amplitude
+  static constexpr float FITAmpFDDA = -1.f;  // FDDA amplitude
+  static constexpr float FITAmpFDDC = -1.f;  // FDDC amplitude
```

Comment:
Use the recommended naming scheme.
Initialise with braces.
What is NDtcoll?

Reply:
ok, NDtcoll is minimum number of bunch crossings around the collision of interest, defined by collision time resolution (σ(tcoll)) over LHC bunch spacing time (∆t_{bc})

Reply:
Not addressed.